lib/scope.js

```diff
@@ -70,6 +81,9 @@ module.exports = {
     let original = source_row.original;
     let var_str = await scope.toTrimmedJSON( scope, { str: original.var });
     let value_str = await scope.toTrimmedJSON( scope, { str: original.value });
```

Comment:
Isn't `original.value` already the secret's name? I might be mixed up on this. I may have been mixed up during the conversation we had the other day. Maybe worth discussing.

It occurs to me now that one potential approach (with some verification that it makes sense before implementing) might be to always use `original.value` whenever we save a variable to a file or to print to the GitHub console. Unless there are specific places where we feel it would be ideal to print as much data as we can.

Reply:
`original.value` isn't the secret's name, it's always the value. 

For the secrets, I could either keep `value` the same, and find everywhere it's printed and change that, or I could change `value` to the secret name, and change everywhere the value was set. I chose the first option, since I thought it was more risky to change the actual test behavior, possibly breaking things if I missed a line somewhere.

Reply:
I see where this confusion is coming from now. It's from what the new function sends to `normalizeTable()`. That's a conundrum because `.original` should probably always be the original input to stay consistent with all the other types of rows. Would it make sense to put the logic in `normalizeTable()`? Add either add the flag as an `options` argument?

Reply:
I see. I guess this makes sense for laying the ground work for having env vars be in tables. But it'll make things a bit harder to make sure the right value is being set in every case.

Reply:
Works fine. Hides a bit of the env var work, but that's probably fine.

Reply:
Just a note for future people: I still agree we necessarily want env vars to be in a story table, but we're still taking advantage of the table to set these variables and so we probably want to keep behavior consistent.